docs/swagger-builder/readme.md

```diff
@@ -99,7 +99,7 @@ To perform validity check of a set of Swagger Specifications, additional tools a
 A code generator is provided to generate an extension of the **Amadeus Digital Experience API**.
 
 ```bash
-yarn yo @ama-sdk/sdk:api-extension
+yarn dlx -p @angular-devkit/schematics-cli schematics @ama-sdk/schematics:api-extension
```

Comment:
Are we sure this is the only part impacted by the sdk-generator/schematics change?

Reply:
No if you know well the topic I would appreciate if you can review this doc :)

Reply:
Let's do it in a another PR